apt/anonymization/anonymizer.py

```diff
@@ -32,7 +35,7 @@ class Anonymize:
     """
 
     def __init__(self, k: int, quasi_identifiers: Union[np.ndarray, list], categorical_features: Optional[list] = None,
-                 is_regression=False):
+                 is_regression=False, train_only_QI=True):
```

Comment:
Please add this param to the docstring. Also, since we didn't do any experiments for the anonymizer, I tend to leave the default as before (i.e., False), even though it's True for minimizer.